Mage.Sets/src/mage/cards/l/LilianaWakerOfTheDead.java

```diff
@@ -0,0 +1,116 @@
+package mage.cards.l;
+
+import mage.abilities.Ability;
+import mage.abilities.LoyaltyAbility;
+import mage.abilities.common.PlaneswalkerEntersWithLoyaltyCountersAbility;
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.abilities.dynamicvalue.common.CardsInControllerGraveyardCount;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.GetEmblemEffect;
+import mage.abilities.effects.common.continuous.BoostTargetEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.cards.Cards;
+import mage.cards.CardsImpl;
+import mage.constants.*;
+import mage.filter.StaticFilters;
+import mage.game.Game;
+import mage.game.command.emblems.LilianaWakerOfTheDeadEmblem;
+import mage.players.Player;
+import mage.target.Target;
+import mage.target.common.TargetCreaturePermanent;
+import mage.target.common.TargetDiscard;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ *
+ * @author htrajan
+ */
+public final class LilianaWakerOfTheDead extends CardImpl {
+
+    private static final DynamicValue xValue = new CardsInControllerGraveyardCount(StaticFilters.FILTER_CARD, -1);
+
+    public LilianaWakerOfTheDead(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.PLANESWALKER}, "{2}{B}{B}");
+        
+        this.addSuperType(SuperType.LEGENDARY);
+        this.subtype.add(SubType.LILIANA);
+        this.addAbility(new PlaneswalkerEntersWithLoyaltyCountersAbility(4));
+
+        // +1: Each player discards a card. Each opponent who can't loses 3 life.
+        this.addAbility(new LoyaltyAbility(new LilianaWakerOfTheDeadDiscardEffect(), 1));
+
+        // −3: Target creature gets -X/-X until end of turn, where X is the number of cards in your graveyard.
+        Ability ability = new LoyaltyAbility(new BoostTargetEffect(
+                xValue, xValue, Duration.EndOfTurn, true
+        ).setText("target creature gets -X/-X until end of turn, where X is the number of cards in your graveyard"), -3);
```

Comment:
Add card hints to inform user about useful data:
![shot_200608_040615](https://user-images.githubusercontent.com/8344157/83983238-b996fa00-a93d-11ea-95fc-49420e9ea9a7.png)

Reply:
oh, yeah good call, added